vignettes/articles/cheatsheet.Rmd

```diff
@@ -0,0 +1,557 @@
+---
+title: "Relational data with dm: Cheat sheet"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{cheatsheet}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+resource_files:
+  - cheatsheet
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  echo = FALSE
+)
+```
+
+[For printing](cheatsheet/cheatsheet-printable.html)
```

Comment:
For now it's printable, not printed.
I think it might be the best solution because it's served at an URL where the font is loaded, whereas when trying to print it automatically I had problems with the font.